tests/torch/test_compressed_graph.py

```diff
@@ -744,15 +744,17 @@ def test_output_quantization(_case_config):
 
 
 def test_custom_quantizable_subgraph_patterns(_case_config):
+    # Decided to remove quantize_patterns option
+    pytest.skip()
     model = test_models.SENet18()
 
     input_shape = [1, 3, 32, 32]
 
     config = get_basic_quantization_config(_case_config.quant_type, input_sample_sizes=input_shape)
 
     config["compression"].update({"quantize_outputs": False,
-                                  "quantizable_subgraph_patterns": [["sigmoid", "__mul__"],
-                                                                    ["__iadd__", "batch_norm"]]})
+                                  "quantize_patterns": [["1 sigmoid", "2 __mul__", "1 -> 2"],
+                                                        ["3 __iadd__", "4 batch_norm", "3 -> 4"]]})
```

Comment:
Better to name this `"fused_patterns"`. 
The node IDs should not be present if possible:

```
"quantize_patterns": [["sigmoid", "__mul__", "sigmoid -> __mul__"],
                                   ["__iadd__", "batch_norm", "__iadd__ -> batch_norm"]]
```
In this case you don't really need exact node IDs, just build each pattern separately. When building a pattern for such representation, if no match attributes are specified for the node, assume that the match should occur on the "type" attribute with a value equal to the node name. Document this as well.

Should also support the following syntax which has no extra lists, if possible:
```
"quantize_patterns": [
"sigmoid, __mul__, sigmoid -> __mul__",
"__iadd__, batch_norm, __iadd__ -> batch_norm"
]

Reply:
Depricated this config file option